pages/community/precice-workshop-2022.md

```diff
@@ -0,0 +1,68 @@
+---
+title: preCICE workshop 2022
+keywords: precice workshop 2022, event, events, workshop
+summary:
+permalink: precice-workshop-2022.html
+toc: false
+redirect_from: /preCICE2022/
+---
+
+<img class="img-responsive center-block" src="images/events/precice2022.svg" alt="preCICE Workshop banner" style="max-width: 500px; margin:auto;">
+
+The 3rd preCICE Workshop will be held virtually through [SimTech](https://www.simtech.uni-stuttgart.de/) and the [University of Stuttgart](https://www.uni-stuttgart.de/en/), from February 21-24, 2022. The workshop is a coming together of the preCICE community to share ideas, experiences and knowledge about using preCICE, and to learn from others in the process.
+
+[Registration](https://precice.org/precice-workshop-2022.html#registration) is now open. We look forward to welcoming back previous participants, and introducing more members to the preCICE community.
```

Comment:
Is this information needed twice? (also below)

Reply:
Are you sure you read the correct link? This one points to the registration section, not to the form.

Reply:
Ah, ok, makes sense :+1: